cv/qc-cv.obo

```diff
@@ -391,6 +390,128 @@ is_a: QC:4000010 ! ID free
 is_a: QC:4000026 ! environment metric
 relationship: has_relation QC:4000013 ! QC metric relation: one run
 
+[Term]
+id: QC:4000069
+name: Fraction of peptide IDs around max MS1
+def: "Fraction of all peptides identified at least X min earlier or later than max MS1 for ID. The values are given in a corresponding list, where the first value is the X minutes before (negative) or after (positive) max MS1 and the second value is the fraction of IDs." [PSI:QC]
```

Comment:
what does `max MS1` mean?
`maximum intensity (i.e. intensity apex) in MS1 of the peptide signal`?

Reply:
changed to:
"Fraction of all peptides identified at least X min earlier or later than the most intense MS1 signal of the identification. The values are given in a corresponding list, where the first value is the X minutes before (negative) or after (positive) max MS1 and the second value is the fraction of IDs."